shared/package.json

```diff
@@ -8,6 +8,8 @@
     "jszip": "^3.7.1",
     "libphonenumber-js": "^1.10.48",
     "lodash": "^4.17.21",
+    "tweetnacl": "^1.0.3",
+    "tweetnacl-util": "^0.15.1",
```

Comment:
@kevin9foong btw we forgot to add these two packages in to `/shared` 💀. 

Finally solved final piece of why chromatic publishing [fails](https://github.com/opengovsg/FormSG/actions/runs/10843484420/job/30090747001) when using vite/rollup. `webpack` is a little looser in resolving imports thus we didn't notice it while `rollup/pnpm` are much stricter.